_topics/en/simplicity.md

```diff
@@ -10,14 +10,31 @@ categories:
 ## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
 excerpt: >
   **Simplicity** is a work in progress low level programming language designed
-  to offer greater functionality than Bitcoin Script. It will require a future
-  soft fork to be enabled on Bitcoin.
+  to offer greater flexibility and expressiveness than Bitcoin Script whilst
+  allowing you to verify the safety, security and costs of your program. It
+  offers native Merklized scripting, formal semantics and type checking.
+  It is not Turing complete but it can verify the execution of Turing complete
+  programs. Although Simplicity provides certain proofs of correctness,
+  sufficient care will need to be applied in the writing of Simplicity programs
+  to ensure they perform as intended. To use Simplicity directly on the
```

Comment:
Maybe instead of saying that care will need to be applied to the writing of Simplicity programs, it's better to note that care still needs to be applied to the design of any contract protocol, whatever language it uses, because execution will occur based only on the code and not on intention.